Formula/jq.rb

```diff
@@ -16,8 +16,7 @@ class Jq < Formula
 
   livecheck do
     url :stable
-    strategy :github_latest
-    regex(%r{href=.*?/tag/jq[._-]v?(\d+(?:\.\d+)+)["' >]}i)
+    regex(/v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^(?:jq[._-])?v?(\d+(?:\.\d+)+)$/i)
```

When version tags generally use the software name as a prefix (e.g., `jq-1.6`) but sometimes omit the prefix (e.g., `1.6rc2`), I prefer to make the prefix optional in the regex. This ensures that the regex only matches these two scenarios and won't match tags for unrelated software in the same repository (e.g., `somethingelse-1.2`). This repository doesn't currently create tags for more than one piece of software but I've seen it in a number of other repositories, so I think it's best to be more explicit (i.e., the open-ended regex can bite us if the situation changes in the future).

Reply:
That's fine; I initially had that but thought it was too complicated.